diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpyta3oihz.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp2aplfk62.cpp
index 4c04dff..17b0256 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpyta3oihz.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp2aplfk62.cpp
@@ -1,8 +1,8 @@
 
             // paddington-ignore
             struct SingleLineIgnored {
-                char a;
                 int b;
+                char a;
             };
             /* paddington-ignore */
             struct BlockCommentIgnored {
