diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpuxufohdg.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpah6mqv5g.cpp
index 24427e0..1ea8176 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpuxufohdg.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpah6mqv5g.cpp
@@ -6,8 +6,8 @@ struct FirstBefore {
 };
 
 struct SecondBefore {
-    char small1;
     int large[35];
+    char small1;
     char small2;
 };
 
