diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp9x684_07.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpph7vq8xp.cpp
index 1d95166..ebe12f2 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp9x684_07.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpph7vq8xp.cpp
@@ -1,9 +1,9 @@
 
             // paddington-ignore
             struct FirstIgnored {
-                char a;
                 int b;
-            };
+            char a;
+                };
             struct ProcessedStruct {
                 char a;
                 int b;
