diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpo4f8ehue.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpgecxeyzk.cpp
index 8bc95d4..0b09ae3 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpo4f8ehue.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpgecxeyzk.cpp
@@ -8,7 +8,7 @@ struct BeforeRegion {
 // paddington-off
 struct OuterRegion {
     char small1;
-    int large[100];
+    char small1;
     char small2;
 };
 
