diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmph1jz_vfp.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpxkinif1g.cpp
index 420b9bb..7c69872 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmph1jz_vfp.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpxkinif1g.cpp
@@ -6,8 +6,8 @@
             };
             /*  paddington-ignore  */
             struct BlockWhitespaceIgnored {
-                char a;
                 int b;
+                char a;
             };
             struct ProcessedStruct {
                 char a;
