diff --git tmp/tmpts_1mvo8.cpp tmp/tmp83tmxhin.cpp
index e6eb78f..086208d 100644
--- tmp/tmpts_1mvo8.cpp
+++ tmp/tmp83tmxhin.cpp
@@ -1,11 +1,11 @@
 
 struct LockedMember {
-    char small1;
-    int locked_field; // paddington-lock
-    char small2;
     int large[50];
+    int locked_field; char small1;
+    char small2;
     char small3;
-};
+// paddington-lock
+    };
 
 int main() {
     LockedMember locked;
