diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp3f5048p1.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpa7o_yjzg.cpp
index 323a733..47d384c 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp3f5048p1.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpa7o_yjzg.cpp
@@ -1,11 +1,9 @@
 
 struct Config {
-    char mode;
-    static int count;
     double value;
-    static const char* name;
     short id;
-};
+char mode;
+    };
 
 int Config::count = 0;
 const char* Config::name = "config";
