diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp1wrwy2xy.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpnsvy3mo7.cpp
index 920b9be..9237298 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp1wrwy2xy.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpnsvy3mo7.cpp
@@ -1,10 +1,10 @@
 
 struct FirstLock {
-    int locked_first; // paddington-lock
+    int locked_first; int large[50];
     char small1;
-    int large[50];
     char small2;
-};
+// paddington-lock
+    };
 
 int main() {
     FirstLock f;
