diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_ya0fr5u.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_pgh0s3k.cpp
index 24427e0..de4c53b 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_ya0fr5u.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_pgh0s3k.cpp
@@ -1,7 +1,7 @@
 
 struct FirstBefore {
-    char small1;
     int large[30];
+    char small1;
     char small2;
 };
 
