diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpv9qzujdq.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmplb7fwndj.cpp
index 920b9be..ec11599 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpv9qzujdq.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmplb7fwndj.cpp
@@ -2,7 +2,7 @@
 struct FirstLock {
     int locked_first; // paddington-lock
     char small1;
-    int large[50];
+    char small1;
     char small2;
 };
 
