diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpo4biok75.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpbaw2p3_n.cpp
index 4c04dff..9386ce3 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpo4biok75.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpbaw2p3_n.cpp
@@ -6,8 +6,8 @@
             };
             /* paddington-ignore */
             struct BlockCommentIgnored {
-                char a;
                 int b;
+                char a;
             };
             struct ProcessedStruct {
                 char a;
