diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpyswgefx5.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpplv8znco.cpp
index 24427e0..38aab98 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpyswgefx5.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpplv8znco.cpp
@@ -20,8 +20,8 @@ struct InRegion {
 // paddington-on
 
 struct FirstAfter {
-    char small1;
     int large[40];
+    char small1;
     char small2;
 };
 
