diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmps26kz_cw.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpuw9_r_cr.cpp
index 1ab7210..c4fbdc3 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmps26kz_cw.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpuw9_r_cr.cpp
@@ -1,9 +1,9 @@
 
 struct Data {
-    char a;
-    friend void access(Data& d);
     int b;
-};
+char a;
+    friend void access(Data& d);
+    };
 
 void access(Data& d) {
     d.a = 'x';
