diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8uqjp078.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_3jktu2q.cpp
index 9fbcf4a..fff4e50 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8uqjp078.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_3jktu2q.cpp
@@ -8,7 +8,7 @@ struct BeforeRegion {
 // paddington-off
 struct FirstInRegion {
     char small1;
-    int large[100];
+    char small1;
     char small2;
 };
 
