diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpyg566lg_.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpx3sxssxm.cpp
index 742238f..fb89f71 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpyg566lg_.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpx3sxssxm.cpp
@@ -6,9 +6,9 @@
             };
             // Paddington-Ignore
             struct MixedCaseIgnored {
-                char a;
                 int b;
-            };
+            char a;
+                };
             struct ProcessedStruct {
                 char a;
                 int b;
