diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpcqakmotq.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8_gu9t4u.cpp
index ab5a36f..2679059 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpcqakmotq.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8_gu9t4u.cpp
@@ -1 +1 @@
-struct Bits { char a; int b : 4; int c : 4; char d; }; int main() { Bits b; return 0; }
\ No newline at end of file
+struct Bits { char a; char d; }; int main() { Bits b; return 0; }
\ No newline at end of file
