diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpjoej90ve.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpvdxxvg_y.cpp
index 51bb82d..9b592e8 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpjoej90ve.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpvdxxvg_y.cpp
@@ -7,8 +7,8 @@ struct BeforeRegion {
 
 // paddington-off
 struct InOffRegion {
-    char small1;
     int large[100];
+    char small1;
     char small2;
 };
 // paddington-on
