diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpftkdi8rp.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpm5cao487.cpp
index 9fbcf4a..3926de0 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpftkdi8rp.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpm5cao487.cpp
@@ -14,7 +14,7 @@ struct FirstInRegion {
 
 struct SecondInRegion {
     char small1;
-    int large[200];
+    char small1;
     char small2;
 };
 // paddington-on
