diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmprfts_9xg.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8cwzyugi.cpp
index 8bc325c..3ed0397 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmprfts_9xg.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8cwzyugi.cpp
@@ -1 +1 @@
-struct Large { char a; long b; char c; long d; char e; long f; }; int main() { Large l; return 0; }
\ No newline at end of file
+struct Large { long b; long d; long f; char a; char c; char e; }; int main() { Large l; return 0; }
\ No newline at end of file
