diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8dwen41d.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpvz9qcsfg.cpp
index 8bc95d4..3897bf3 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8dwen41d.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpvz9qcsfg.cpp
@@ -7,8 +7,8 @@ struct BeforeRegion {
 
 // paddington-off
 struct OuterRegion {
-    char small1;
     int large[100];
+    char small1;
     char small2;
 };
 
