diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp7u317_c8.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp4qzs7jqf.cpp
index 7ac341a..e3b4782 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp7u317_c8.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp4qzs7jqf.cpp
@@ -1,7 +1,7 @@
 
 struct SingleLock {
-    char small1;
     int locked_field; // paddington-lock
+    char small1;
     char small2;
 };
 
