diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpky6n6hnf.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpsxvmuw3p.cpp
index 742238f..55faba6 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpky6n6hnf.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpsxvmuw3p.cpp
@@ -1,9 +1,9 @@
 
             // PADDINGTON-IGNORE
             struct UpperCaseIgnored {
-                char a;
                 int b;
-            };
+            char a;
+                };
             // Paddington-Ignore
             struct MixedCaseIgnored {
                 char a;
