diff --git tmp/tmphrigvcdy.cpp tmp/tmps38r_bc3.cpp
index eb4efc9..2acc498 100644
--- tmp/tmphrigvcdy.cpp
+++ tmp/tmps38r_bc3.cpp
@@ -1,11 +1,11 @@
 
 struct MultipleLocks {
-    char small1;
-    int locked1; // paddington-lock
+    int locked1; int locked2; char small1;
     char small2;
-    int locked2; // paddington-lock
     char small3;
-};
+// paddington-lock
+    // paddington-lock
+    };
 
 int main() {
     MultipleLocks m;
