diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpw7f3315l.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp6eiyklqt.cpp
index 420b9bb..c9ddb27 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpw7f3315l.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp6eiyklqt.cpp
@@ -10,9 +10,9 @@
                 int b;
             };
             struct ProcessedStruct {
-                char a;
                 int b;
-            };
+            char a;
+                };
             int main() {
                 WhitespaceIgnored whitespace;
                 BlockWhitespaceIgnored blockWhitespace;
