diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpjh2tx0wz.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp7yva_0i2.cpp
index 5983060..4d7897c 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpjh2tx0wz.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp7yva_0i2.cpp
@@ -4,8 +4,8 @@
                 int b;
             };
             struct Derived : Base {
-                char c;
                 double d;
-            };
+            char c;
+                };
             int main() { Derived d; return 0; }
             
\ No newline at end of file
