diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp4u27pi8k.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmplu0uyufb.cpp
index 420b9bb..f934305 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp4u27pi8k.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmplu0uyufb.cpp
@@ -1,9 +1,9 @@
 
             //   paddington-ignore   
             struct WhitespaceIgnored {
-                char a;
                 int b;
-            };
+            char a;
+                };
             /*  paddington-ignore  */
             struct BlockWhitespaceIgnored {
                 char a;
