diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpn8m8gpti.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmptu_25d57.cpp
index 8bc95d4..c4c94ec 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpn8m8gpti.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmptu_25d57.cpp
@@ -21,8 +21,8 @@ struct InnerRegion {
 // paddington-on
 
 struct StillInOuter {
-    char small1;
     int large[150];
+    char small1;
     char small2;
 };
 // paddington-on
