diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpoz0j9puo.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp59gtfvv1.cpp
index 2b041ca..ba5788b 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpoz0j9puo.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp59gtfvv1.cpp
@@ -1 +1 @@
-struct Mixed { char a; double b; short c; int d; }; int main() { Mixed m; return 0; }
\ No newline at end of file
+struct Mixed { double b; int d; short c; char a; }; int main() { Mixed m; return 0; }
\ No newline at end of file
