diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp7uahblzo.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpq_s7df4q.cpp
index 420b9bb..ab967d8 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp7uahblzo.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpq_s7df4q.cpp
@@ -1,8 +1,8 @@
 
             //   paddington-ignore   
             struct WhitespaceIgnored {
-                char a;
                 int b;
+                char a;
             };
             /*  paddington-ignore  */
             struct BlockWhitespaceIgnored {
