diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpp_860b0h.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmphi1r_0pt.cpp
index 9fbcf4a..82836ea 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpp_860b0h.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmphi1r_0pt.cpp
@@ -7,8 +7,8 @@ struct BeforeRegion {
 
 // paddington-off
 struct FirstInRegion {
-    char small1;
     int large[100];
+    char small1;
     char small2;
 };
 
