diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmppndlxn6p.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpd2dqutco.cpp
index ace835b..725c07f 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmppndlxn6p.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpd2dqutco.cpp
@@ -1,8 +1,8 @@
 
             // paddington-ignore
             struct IgnoredStruct {
-                char a;
                 int b;
+                char a;
             };
             struct ProcessedStruct {
                 char a;
