diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpj4c6du25.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp51oo5md_.cpp
index 4c04dff..3f3b57c 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpj4c6du25.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp51oo5md_.cpp
@@ -1,9 +1,9 @@
 
             // paddington-ignore
             struct SingleLineIgnored {
-                char a;
                 int b;
-            };
+            char a;
+                };
             /* paddington-ignore */
             struct BlockCommentIgnored {
                 char a;
