diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpkgmwgd_s.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmptqwjim27.cpp
index b83699b..4607e8d 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpkgmwgd_s.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmptqwjim27.cpp
@@ -1 +1 @@
-struct Inner { char x; int y; }; struct Outer { char a; Inner b; char c; }; int main() { Outer o; return 0; }
\ No newline at end of file
+struct Inner { int y; char x; }; struct Outer { char a; Inner b; char c; }; int main() { Outer o; return 0; }
\ No newline at end of file
