diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpmyy04umi.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpu5fmrmyg.cpp
index 1d95166..ad10c0b 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpmyy04umi.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpu5fmrmyg.cpp
@@ -1,8 +1,8 @@
 
             // paddington-ignore
             struct FirstIgnored {
-                char a;
                 int b;
+                char a;
             };
             struct ProcessedStruct {
                 char a;
