diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp5k7wbkdq.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp6y1ujl3e.cpp
index cb6de31..2972601 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp5k7wbkdq.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp6y1ujl3e.cpp
@@ -8,7 +8,7 @@ struct FirstStruct {
 
 struct SecondStruct {
     char small1;
-    int large[200];
+    char small1;
     char small2;
 };
 
