diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmprf1ess29.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpxshvt9l5.cpp
index 815d49e..d17315e 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmprf1ess29.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpxshvt9l5.cpp
@@ -1,8 +1,8 @@
 
             struct Data {
-                char a;  // first member
-                int b;   // second member
-                char c;  // third member
+                int b;   char a;  char c;  // first member
+                // second member
+                // third member
             };
             int main() {
                 Data d;  // instance
