diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpisi16nz1.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpk_dhi0qz.cpp
index 815d49e..f145d3c 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpisi16nz1.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpk_dhi0qz.cpp
@@ -1,7 +1,7 @@
 
             struct Data {
-                char a;  // first member
                 int b;   // second member
+                char a;  // first member
                 char c;  // third member
             };
             int main() {
