diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpv1rrjaws.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpkej2yilp.cpp
index ace835b..15c823f 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpv1rrjaws.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpkej2yilp.cpp
@@ -1,9 +1,9 @@
 
             // paddington-ignore
             struct IgnoredStruct {
-                char a;
                 int b;
-            };
+            char a;
+                };
             struct ProcessedStruct {
                 char a;
                 int b;
