diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp5po1ezsi.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpkvb01eba.cpp
index fef37da..0cb729e 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp5po1ezsi.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpkvb01eba.cpp
@@ -4,8 +4,8 @@
                 int y;
             };
             struct Outer {
-                char a;
                 Inner inner;
+                char a;
                 char b;
             };
             int main() {
