diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmps336urmk.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpkiv8zekd.cpp
index cb6de31..fe4bab5 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmps336urmk.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpkiv8zekd.cpp
@@ -2,7 +2,7 @@
 // paddington-off
 struct FirstStruct {
     char small1;
-    int large[100];
+    char small1;
     char small2;
 };
 
