diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp01q767mt.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp3o5ub_x1.cpp
index f6b91af..101f147 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp01q767mt.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp3o5ub_x1.cpp
@@ -1,8 +1,8 @@
 
             #define TYPE int
             struct MacroData {
-                char a;
                 TYPE b;
+                char a;
                 char c;
             };
             int main() {
