diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpyy4d63jx.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpvc13g330.cpp
index 420b9bb..b8acc48 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpyy4d63jx.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpvc13g330.cpp
@@ -6,9 +6,9 @@
             };
             /*  paddington-ignore  */
             struct BlockWhitespaceIgnored {
-                char a;
                 int b;
-            };
+            char a;
+                };
             struct ProcessedStruct {
                 char a;
                 int b;
