diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpm7m3pymq.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpn6k1o9bj.cpp
index 7ac341a..5ed5fc3 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpm7m3pymq.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpn6k1o9bj.cpp
@@ -1,9 +1,9 @@
 
 struct SingleLock {
-    char small1;
-    int locked_field; // paddington-lock
+    int locked_field; char small1;
     char small2;
-};
+// paddington-lock
+    };
 
 int main() {
     SingleLock s;
