diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpyu4yh0h_.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpgdl3pzed.cpp
index cb6de31..9b7833f 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpyu4yh0h_.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpgdl3pzed.cpp
@@ -7,8 +7,8 @@ struct FirstStruct {
 };
 
 struct SecondStruct {
-    char small1;
     int large[200];
+    char small1;
     char small2;
 };
 
