diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpis_c9a81.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpl6y64uyc.cpp
index 742238f..0529615 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpis_c9a81.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpl6y64uyc.cpp
@@ -6,8 +6,8 @@
             };
             // Paddington-Ignore
             struct MixedCaseIgnored {
-                char a;
                 int b;
+                char a;
             };
             struct ProcessedStruct {
                 char a;
