diff --git tmp/tmpd3drlrdh.cpp tmp/tmpdtsikom8.cpp
index 323a733..b6fc21a 100644
--- tmp/tmpd3drlrdh.cpp
+++ tmp/tmpdtsikom8.cpp
@@ -1,11 +1,11 @@
 
 struct Config {
-    char mode;
-    static int count;
     double value;
-    static const char* name;
     short id;
-};
+char mode;
+    static int count;
+    static const char* name;
+    };
 
 int Config::count = 0;
 const char* Config::name = "config";
