diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmplgnybc1n.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp0c1rzict.cpp
index 24427e0..8428007 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmplgnybc1n.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp0c1rzict.cpp
@@ -27,7 +27,7 @@ struct FirstAfter {
 
 struct SecondAfter {
     char small1;
-    int large[45];
+    char small1;
     char small2;
 };
 
