diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpq2q_6swy.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpeiv5qsyv.cpp
index 742238f..11385d8 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpq2q_6swy.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpeiv5qsyv.cpp
@@ -1,8 +1,8 @@
 
             // PADDINGTON-IGNORE
             struct UpperCaseIgnored {
-                char a;
                 int b;
+                char a;
             };
             // Paddington-Ignore
             struct MixedCaseIgnored {
