diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpwl6leglx.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpod797x5n.cpp
index 9fbcf4a..c279c38 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpwl6leglx.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpod797x5n.cpp
@@ -13,8 +13,8 @@ struct FirstInRegion {
 };
 
 struct SecondInRegion {
-    char small1;
     int large[200];
+    char small1;
     char small2;
 };
 // paddington-on
