diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpysv52255.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpi180tcp7.cpp
index 1a1218f..f4e7afb 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpysv52255.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpi180tcp7.cpp
@@ -1 +1 @@
-struct S { char a; int b; }; union U { S s; long l; }; int main() { U u; return 0; }
\ No newline at end of file
+struct S { int b; char a; }; union U { S s; long l; }; int main() { U u; return 0; }
\ No newline at end of file
