diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmprynzty74.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_8ibkskl.cpp
index b83699b..a46c3d8 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmprynzty74.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_8ibkskl.cpp
@@ -1 +1 @@
-struct Inner { char x; int y; }; struct Outer { char a; Inner b; char c; }; int main() { Outer o; return 0; }
\ No newline at end of file
+struct Inner { char x; int y; }; struct Outer { Inner b; char a; char c; }; int main() { Outer o; return 0; }
\ No newline at end of file
