diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp1888630t.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpku4xmo31.cpp
index 1d95166..1bf56bd 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp1888630t.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpku4xmo31.cpp
@@ -10,9 +10,9 @@
             };
             // paddington-ignore
             struct SecondIgnored {
-                char a;
                 int b;
-            };
+            char a;
+                };
             int main() {
                 FirstIgnored first;
                 ProcessedStruct processed;
