diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpws8t6paz.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpz_rp8hx5.cpp
index 81eaf96..47a6d17 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpws8t6paz.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpz_rp8hx5.cpp
@@ -1 +1 @@
-struct Free { char a; int b; char c; Free(int x, char y, char z) : a(y), b(x), c(z) {} }; int main() { Free f(42, 'x', 'y'); return 0; }
\ No newline at end of file
+struct Free { int b; char a; char c; Free(int x, char y, char z) : a(y), b(x), c(z) {} }; int main() { Free f(42, 'x', 'y'); return 0; }
\ No newline at end of file
