diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp52lbilra.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpmkm5bnmu.cpp
index eb4efc9..2acc498 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp52lbilra.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpmkm5bnmu.cpp
@@ -1,11 +1,11 @@
 
 struct MultipleLocks {
-    char small1;
-    int locked1; // paddington-lock
+    int locked1; int locked2; char small1;
     char small2;
-    int locked2; // paddington-lock
     char small3;
-};
+// paddington-lock
+    // paddington-lock
+    };
 
 int main() {
     MultipleLocks m;
