diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp78gitojh.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpfwxy2r4x.cpp
index 6134502..cb99dcd 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp78gitojh.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpfwxy2r4x.cpp
@@ -1 +1 @@
-struct Ptrs { char a; void* b; char c; void* d; }; int main() { Ptrs p; return 0; }
\ No newline at end of file
+struct Ptrs { void* b; void* d; char a; char c; }; int main() { Ptrs p; return 0; }
\ No newline at end of file
