diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmph467aho1.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp0crgftbn.cpp
index 6d0f0b6..daa39cc 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmph467aho1.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp0crgftbn.cpp
@@ -8,7 +8,7 @@ struct IgnoredStruct {
 
 struct NormalStruct {
     char small1;
-    int large[50];
+    char small1;
     char small2;
 };
 
