diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmprgwe_moo.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpz37zmdrd.cpp
index 4c04dff..b1ad742 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmprgwe_moo.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpz37zmdrd.cpp
@@ -6,9 +6,9 @@
             };
             /* paddington-ignore */
             struct BlockCommentIgnored {
-                char a;
                 int b;
-            };
+            char a;
+                };
             struct ProcessedStruct {
                 char a;
                 int b;
