diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8mij59pb.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpubtjtxn6.cpp
index e6eb78f..842d254 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8mij59pb.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpubtjtxn6.cpp
@@ -1,11 +1,11 @@
 
 struct LockedMember {
+    int locked_field; int large[50];
     char small1;
-    int locked_field; // paddington-lock
     char small2;
-    int large[50];
     char small3;
-};
+// paddington-lock
+    };
 
 int main() {
     LockedMember locked;
