diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpbowgpe_u.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8nktealz.cpp
index 5983060..089f48b 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpbowgpe_u.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp8nktealz.cpp
@@ -1,8 +1,8 @@
 
             struct Base {
-                char a;
                 int b;
-            };
+            char a;
+                };
             struct Derived : Base {
                 char c;
                 double d;
