diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp4n7i2hr3.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpg97e9l1_.cpp
index cb6de31..9b7833f 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp4n7i2hr3.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpg97e9l1_.cpp
@@ -7,8 +7,8 @@ struct FirstStruct {
 };
 
 struct SecondStruct {
-    char small1;
     int large[200];
+    char small1;
     char small2;
 };
 
