diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpji16802_.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpjq_u02ni.cpp
index 8f2cef6..6221f15 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpji16802_.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpjq_u02ni.cpp
@@ -1,7 +1,7 @@
 
             struct Data {
-                char a;
                 int b;
+                char a;
                 char c;
             };
             int main() {
