diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp2k_z95lm.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpswoyh2gc.cpp
index 420b9bb..b9fa2fc 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp2k_z95lm.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpswoyh2gc.cpp
@@ -10,8 +10,8 @@
                 int b;
             };
             struct ProcessedStruct {
-                char a;
                 int b;
+                char a;
             };
             int main() {
                 WhitespaceIgnored whitespace;
