diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp1wqhhp_2.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_ndxdzok.cpp
index 8bc95d4..36a3211 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp1wqhhp_2.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_ndxdzok.cpp
@@ -1,7 +1,7 @@
 
 struct BeforeRegion {
-    char small1;
     int large[30];
+    char small1;
     char small2;
 };
 
