diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp75neltgg.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp7qrowkf1.cpp
index 617a21b..09ec0d3 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp75neltgg.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp7qrowkf1.cpp
@@ -1 +1 @@
-struct Arrays { char a; int b[2]; char c; int d[2]; }; int main() { Arrays arr; return 0; }
\ No newline at end of file
+struct Arrays { int b[2]; int d[2]; char a; char c; }; int main() { Arrays arr; return 0; }
\ No newline at end of file
