diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpdnf_62v4.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmphfhaw12p.cpp
index 8bc95d4..d5761b0 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpdnf_62v4.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmphfhaw12p.cpp
@@ -15,7 +15,7 @@ struct OuterRegion {
 // paddington-off
 struct InnerRegion {
     char small1;
-    int large[200];
+    char small1;
     char small2;
 };
 // paddington-on
