diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpy4lg4wo7.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmppq7xsmdr.cpp
index b338cd8..a6fe1f7 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpy4lg4wo7.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmppq7xsmdr.cpp
@@ -1 +1 @@
-struct A { char a; int b; }; int main() { A a; return 0; }
\ No newline at end of file
+struct A { int b; char a; }; int main() { A a; return 0; }
\ No newline at end of file
