diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp3m5nb38k.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpr0ypi29h.cpp
index eb4efc9..f8ce905 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp3m5nb38k.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpr0ypi29h.cpp
@@ -1,9 +1,9 @@
 
 struct MultipleLocks {
-    char small1;
     int locked1; // paddington-lock
-    char small2;
     int locked2; // paddington-lock
+    char small1;
+    char small2;
     char small3;
 };
 
