diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmps1dbol16.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpdhhb0fr4.cpp
index 742238f..11385d8 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmps1dbol16.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpdhhb0fr4.cpp
@@ -1,8 +1,8 @@
 
             // PADDINGTON-IGNORE
             struct UpperCaseIgnored {
-                char a;
                 int b;
+                char a;
             };
             // Paddington-Ignore
             struct MixedCaseIgnored {
