diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpxtn6fi5g.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmppaj2kwjt.cpp
index fef37da..b804515 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpxtn6fi5g.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmppaj2kwjt.cpp
@@ -1,8 +1,8 @@
 
             struct Inner {
-                char x;
                 int y;
-            };
+            char x;
+                };
             struct Outer {
                 char a;
                 Inner inner;
