diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_zx53phl.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpdrw4amzy.cpp
index ded2a44..f851253 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmp_zx53phl.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpdrw4amzy.cpp
@@ -1,11 +1,11 @@
 
 struct MiddleLock {
-    char small1;
     int large1[30];
-    int locked_middle; // paddington-lock
-    int large2[30];
+    int locked_middle; int large2[30];
+    char small1;
     char small2;
-};
+// paddington-lock
+    };
 
 int main() {
     MiddleLock m;
