diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpi_e6sp6b.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmptqpilema.cpp
index 24427e0..eb9ae75 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpi_e6sp6b.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmptqpilema.cpp
@@ -7,7 +7,7 @@ struct FirstBefore {
 
 struct SecondBefore {
     char small1;
-    int large[35];
+    char small1;
     char small2;
 };
 
