diff --git var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpuemx1fam.cpp var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmps57ns9wk.cpp
index 6d0f0b6..d0ae289 100644
--- var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmpuemx1fam.cpp
+++ var/folders/vn/llg2yrpd6q17y53ny2f6yfyw0000gq/T/tmps57ns9wk.cpp
@@ -7,8 +7,8 @@ struct IgnoredStruct {
 };
 
 struct NormalStruct {
-    char small1;
     int large[50];
+    char small1;
     char small2;
 };
 
